test/parallel/test-http-set-trailers.js

```diff
@@ -68,9 +68,9 @@ function testHttp11(port, callback) {
     res_buffer += chunk;
     if (/0\r\n/.test(res_buffer)) { // got the end.
       clearTimeout(tid);
-      assert.ok(
-        /0\r\nx-foo: bar\r\n\r\n$/.test(res_buffer),
-        `No trailer in HTTP/1.1 response. Response buffer: ${res_buffer}`
+      assert.match(
+        res_buffer,
+        /0\r\nx-foo: bar\r\n\r\n$/,
```

Comment:
Is there a reason to remove the messages? The `AssertionError`s already have the regex and tested string in `expected` and `actual` properties, making default message not more informative.
```suggestion
      assert.match(
        res_buffer,
        /0\r\nx-foo: bar\r\n\r\n$/,
        'No trailer in HTTP/1.1 response',
```

Reply:
These rules seem to be originated from https://github.com/nodejs/node/pull/22849 which restricts string literals because they mask the compared values. Unless I'm missing something, in modern versions of `strictEqual()` nothing gets masked, omitting message just replaces it with `Expected values to be strictly equal:` while values are printed anyway. IMHO these rules might be outdated.

For the sake of consistency, we can align it with https://github.com/nodejs/node/pull/22849#issuecomment-421479453, making message into comment:
```suggestion
      // Ensure trailer being in HTTP/1.1 response
      assert.match(
        res_buffer,
        /0\r\nx-foo: bar\r\n\r\n$/,
```

(on a side note, the original message in this particular one was a template literal so it wouldn't break linter rule anyway 😄)